files/CooperHewitt-Book.ufo/fontinfo.plist

```diff
@@ -126,7 +126,7 @@
 	<key>openTypeOS2VendorID</key>
 	<string>VLLG</string>
 	<key>openTypeOS2WeightClass</key>
-	<integer>705</integer>
+	<integer>300</integer>
```

Comment:
OK. About Book possibly being slightly thicker than Regular if different, it depends whom you ask: http://www.webtype.com/info/articles/fonts-weights/